src/ws/hooks/item.test.ts

```diff
@@ -78,14 +80,14 @@ describe('Ws Item Hooks', () => {
       const itemEvent = {
         kind: 'kind',
         op: OPS.UPDATE,
-        item: newItem,
+        item: JSON.parse(JSON.stringify(newItemRecord.toJS())),
```

Comment:
Is it necessary to deep clone? Won't the `toJS()` create a new object?

Reply:
This is done to explicitly convert the `Date` to their serialized counter part. I am not trying to create a deep clone here.